src/plugins/workspace/public/components/workspace_form/workspace_detail_form.tsx

```diff
@@ -73,28 +72,10 @@ export const WorkspaceDetailForm = (props: WorkspaceDetailedFormProps) => {
     defaultValues?.permissionSettings?.map((item) => item.id) ?? []
   );
 
-  const [isSaving, setIsSaving] = useState(false);
-
-  // Handle beforeunload event
-  useEffect(() => {
-    const handleBeforeUnload = (event: BeforeUnloadEvent) => {
-      if (!isSaving && isEditing && numberOfChanges > 0) {
-        event.preventDefault();
-      }
-    };
-
-    window.addEventListener('beforeunload', handleBeforeUnload);
-
-    return () => {
-      window.removeEventListener('beforeunload', handleBeforeUnload);
-    };
-  }, [isEditing, isSaving, numberOfChanges]);
```

Comment:
It is used when user tries to force reload the page through browser, should we keep it?

Reply:
No, the `onAppLeave` will cover this case.

Reply:
Oh, get it, thanks.